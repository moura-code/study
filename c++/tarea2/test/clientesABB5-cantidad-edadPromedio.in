crearClientesABB
cantidadClientesClientesABB
edadPromedioClientesABB
crearCliente 10 Artemio Lopez 34
agregarAClientesABB
cantidadClientesClientesABB
edadPromedioClientesABB
crearCliente 11 Casimira Rodriguez 22
agregarAClientesABB
cantidadClientesClientesABB
edadPromedioClientesABB
crearCliente 12 Zenobia Lopez 40
agregarAClientesABB
cantidadClientesClientesABB
edadPromedioClientesABB
crearCliente 13 Esmeralda Perez 16
agregarAClientesABB
cantidadClientesClientesABB
edadPromedioClientesABB
liberarClientesABB
Fin
