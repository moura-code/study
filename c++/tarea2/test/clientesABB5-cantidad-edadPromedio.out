Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> El clientesABB ha sido creado de forma exitosa.
2> La cantidad de clientes en el clientesABB es 0.
3> La edad promedio del clientesABB es 0.00.
4> Cliente creado de forma exitosa.
5> Se ha agregado el cliente al clientesABB de forma exitosa.
6> La cantidad de clientes en el clientesABB es 1.
7> La edad promedio del clientesABB es 34.00.
8> Cliente creado de forma exitosa.
9> Se ha agregado el cliente al clientesABB de forma exitosa.
10> La cantidad de clientes en el clientesABB es 2.
11> La edad promedio del clientesABB es 28.00.
12> Cliente creado de forma exitosa.
13> Se ha agregado el cliente al clientesABB de forma exitosa.
14> La cantidad de clientes en el clientesABB es 3.
15> La edad promedio del clientesABB es 32.00.
16> Cliente creado de forma exitosa.
17> Se ha agregado el cliente al clientesABB de forma exitosa.
18> La cantidad de clientes en el clientesABB es 4.
19> La edad promedio del clientesABB es 28.00.
20> ClientesABB liberado con exito.
21> Fin.
